Imshk.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Shashank Dwivedi",
+  "type": "personal",
+  "bio": "Student, Full stack developer, Blogger",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Shashankoffice",
+      "icon": "FaTwitter"
+  },
+    {"name": "Follow me on Github ",
+      "url": "https://github.com/Imshk",
+      "icon": "FaGithub"
+    },
+    {"name": "Follow me on linkedIn ",
+      "url": "https://www.linkedin.com/in/shashankoffice/",
+      "icon": "FaGithub"
```

Comment:
icon value should be **FaLinkedin** . please update!

Reply:
Thank you @Hrishi5111998 , I have updated .